crates/contracts/build.rs

```diff
@@ -205,6 +205,8 @@ fn main() {
     });
     generate_contract("IUniswapLikeRouter");
     generate_contract("IUniswapLikePair");
+    // EIP-1271 contract - SignatureValidator
+    generate_contract("EIP1271SignatureValidator");
```

Comment:
ubernit: Can we rename this to `ERC1271...` as its officially [an ERC now](https://eips.ethereum.org/EIPS/eip-1271#specification)

Reply:
Will do.

Reply:
Renamed